### ISSUE TITLE
QDataTables failed to install

### ISSUE THREAD

**Issue by kukrik** [qcubed/plugin_datatables#11]
Here is the error message that you must do to get it to work? I used exactly the command line:
```
composer require datatables/datatables qcubed/plugin_datatables
```
Thanks in advance!

```
..........:qcubed-shannon Papikodu$ sudo composer require datatables/datatables qcubed/plugin_datatables
Using version ^1.10 for datatables/datatables
Using version ^1.0 for qcubed/plugin_datatables
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: datatables/datatables[dev-master, 1.10.0].
    - Can only install one of: datatables/datatables[dev-master, 1.10.0-beta.1].
    - Can only install one of: datatables/datatables[dev-master, 1.10.0-beta.2].
    - Can only install one of: datatables/datatables[dev-master, 1.10.0-rc.1].
    - Can only install one of: datatables/datatables[dev-master, 1.10.1].
    - Can only install one of: datatables/datatables[dev-master, 1.10.10].
    - Can only install one of: datatables/datatables[dev-master, 1.10.2].
    - Can only install one of: datatables/datatables[dev-master, 1.10.3].
    - Can only install one of: datatables/datatables[dev-master, 1.10.4].
    - Can only install one of: datatables/datatables[dev-master, 1.10.5].
    - Can only install one of: datatables/datatables[dev-master, 1.10.6].
    - Can only install one of: datatables/datatables[dev-master, 1.10.7].
    - Can only install one of: datatables/datatables[dev-master, 1.10.8].
    - Can only install one of: datatables/datatables[dev-master, 1.10.9].
    - qcubed/plugin_datatables v1.0.0 requires datatables/datatables dev-master -> satisfiable by datatables/datatables[dev-master].
    - Installation request for qcubed/plugin_datatables ^1.0 -> satisfiable by qcubed/plugin_datatables[v1.0.0].
    - Installation request for datatables/datatables ^1.10 -> satisfiable by datatables/datatables[1.10.0, 1.10.0-beta.1, 1.10.0-beta.2, 1.10.0-rc.1, 1.10.1, 1.10.10, 1.10.2, 1.10.3, 1.10.4, 1.10.5, 1.10.6, 1.10.7, 1.10.8, 1.10.9].

Installation failed, reverting ./composer.json to its original content.
```

**Comment by spekary** [comment]
Primarily a problem with version numbers. You can fix it by specifying dev-master for the versions for both. I will fix it so the default installation instructions should still work.

**Comment by spekary** [comment]
Try again, use the same installation line:
```
composer require datatables/datatables qcubed/plugin_datatables
```

**Comment by kukrik** [comment]
:+1: installation works. But I had a long time off, I can not quickly hit with an error: Undefined variable in a GET or property 'QDataTable' class: FilterString.

I lay down screenshots
<img width="1280" alt="screen shot 2015-12-13 at 21 25 28" src="https://cloud.githubusercontent.com/assets/2853034/11768791/7881dda0-a1e0-11e5-8a5c-8e4593d54bf9.png">

**Comment by spekary** [comment]
Thanks for the report. Fixed in #12
